BitbucketPS.psm1

```diff
@@ -0,0 +1,489 @@
+function ConvertFrom-Json2
```

Comment:
What is going on here?

Why are the functions being declared in the psm1 file?
psm1 file was generic already (but should be updated to this: https://github.com/lipkau/JiraPS/blob/enhance/harmonizeStyling/JiraPS/JiraPS.psm1)

Reply:
I dislike that approach, it causes slower module loads especially if someone places the module on a network share.  By putting the functions in the PSM file they load almost instantly.

Reply:
I understand your preference.
However, this is a community managed repo and we have established the loading of dedicated files as the standard in order to make the maintenance and readability easier.